tests/labhub_test.py

```diff
@@ -368,22 +385,28 @@ def test_mark_cmd(self):
         mock_gitlab_mr.labels = ['process/wip']
         testbot.assertCommand(cmd_github.format('pending', 'coala', 'a', '23'),
                               'marked pending review')
-        testbot.assertCommand(cmd_github.format('pending-review', 'coala', 'a', '23'),
+        testbot.assertCommand(cmd_github.format('pending-review', 'coala',
+                                                'a', '23'),
                               'marked pending review')
-        testbot.assertCommand(cmd_github.format('pending review', 'coala', 'a', '23'),
+        testbot.assertCommand(cmd_github.format('pending review',
+                                                'coala', 'a', '23'),
                               'marked pending review')
+        # Stop ignoring
 
     def test_alive(self):
         labhub, testbot = plugin_testbot(plugins.labhub.LabHub, logging.ERROR)
         with patch('plugins.labhub.time.sleep') as mock_sleep:
             labhub.gh_repos = {
                 'coala': create_autospec(IGitt.GitHub.GitHub.GitHubRepository),
-                'coala-bears': create_autospec(IGitt.GitHub.GitHub.GitHubRepository),
-                'coala-utils': create_autospec(IGitt.GitHub.GitHub.GitHubRepository)
+                'coala-bears': create_autospec(
```

Comment:
drop `create_autospec` to next line and indent only four from the start of the key.